src/compiler/parser.ts

```diff
@@ -6178,76 +6168,133 @@ namespace ts {
                 Debug.assert(end <= content.length);
 
                 let tags: NodeArray<JSDocTag>;
-                let result: JSDocComment;
+                const comments: string[] = [];
+                let result: JSDoc;
 
                 // Check for /** (JSDoc opening part)
-                if (content.charCodeAt(start) === CharacterCodes.slash &&
-                    content.charCodeAt(start + 1) === CharacterCodes.asterisk &&
-                    content.charCodeAt(start + 2) === CharacterCodes.asterisk &&
-                    content.charCodeAt(start + 3) !== CharacterCodes.asterisk) {
-
+                if (!isJsDocStart(content, start)) {
+                    return result;
+                }
 
-                    // + 3 for leading /**, - 5 in total for /** */
-                    scanner.scanRange(start + 3, length - 5, () => {
-                        // Initially we can parse out a tag.  We also have seen a starting asterisk.
-                        // This is so that /** * @type */ doesn't parse.
-                        let canParseTag = true;
-                        let seenAsterisk = true;
+                // + 3 for leading /**, - 5 in total for /** */
+                scanner.scanRange(start + 3, length - 5, () => {
+                    // Initially we can parse out a tag.  We also have seen a starting asterisk.
+                    // This is so that /** * @type */ doesn't parse.
+                    let advanceToken = true;
+                    let state = JSDocState.SawAsterisk;
+                    let margin: number | undefined = undefined;
+                    let indent = start - Math.max(content.lastIndexOf("\n", start), 0) + 4;

```

Comment:
Oh. That part is embarassing. `4 === '/** '.length`, basically. I should add a comment like the one on line 6179